implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -218,9 +222,13 @@ private double minFromHistogramSnapshot(HistogramSnapshot histogramSnapshot, Tim
     }
 
     Stream<String> toFunctionTimerLine(FunctionTimer meter) {
+        long count = Double.valueOf(meter.count()).longValue();
```

Comment:
I think you could use a simple cast here, right? :thinking: 
```suggestion
        long count = (long) meter.count();
```